[2025-10-16T10:20:09.868+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-16T10:20:09.887+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:19:00+00:00 [queued]>
[2025-10-16T10:20:09.895+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:19:00+00:00 [queued]>
[2025-10-16T10:20:10.808+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-16T10:20:11.440+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dag2_bash_1> on 2025-10-16 10:19:00+00:00
[2025-10-16T10:20:11.448+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=332) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-16T10:20:11.449+0000] {standard_task_runner.py:63} INFO - Started process 334 to run task
[2025-10-16T10:20:11.449+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'dag2_bash_1', 'scheduled__2025-10-16T10:19:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmp4jm2qgdj']
[2025-10-16T10:20:11.452+0000] {standard_task_runner.py:91} INFO - Job 84: Subtask dag2_bash_1
[2025-10-16T10:20:12.893+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-16T10:20:12.927+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:19:00+00:00 [running]> on host bb861ccd12fb
[2025-10-16T10:20:15.194+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='dag2_bash_1' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T10:19:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-16T10:19:00+00:00'
[2025-10-16T10:20:15.196+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-16T10:20:15.197+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-16T10:20:15.198+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hello bash"']
[2025-10-16T10:20:15.209+0000] {subprocess.py:86} INFO - Output:
[2025-10-16T10:20:15.210+0000] {subprocess.py:93} INFO - hello bash
[2025-10-16T10:20:15.211+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-16T10:20:15.212+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-16T10:20:16.814+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=dag2_bash_1, run_id=scheduled__2025-10-16T10:19:00+00:00, execution_date=20251016T101900, start_date=20251016T102009, end_date=20251016T102016
[2025-10-16T10:20:18.414+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-16T10:20:18.430+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-16T10:20:19.706+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
